docs/install/install-local-migrate.md

```diff
@@ -83,15 +83,6 @@ Prior to attempting this step, do consider the following:
   `mysqldump -u username -p database_name > prod_drupal_site_082019.sql`
   * Copy this file down to your local laptop or workstation.
 
-#### Fedora hash size (Conditional)
```

Comment:
Just double checking that this section should be removed.   Is this not needed on a migrate from an existing (non-ISLE) islandora system into ISLE?

Reply:
Oh, I see what you've done now.  Because someone might not view the troubleshooting sheet, I'd put at least a reference to that section here. It's something folks should be told about and I wouldn't want them to miss this important detail.  My  $.02.  :)

Reply:
Excellent point. I had meant to do this... really :)
Now it's done.